tests/teal/fac_by_ref_expected.teal

```diff
@@ -0,0 +1,41 @@
+#pragma version 6          Case n = 1   Case n = 2
+int 42                      >1          >2
+store 0                     0:1         0:2
+int 0                       >0          >0
+callsub factorial_0         CALL        CALL
+load 0                      >1          >2   
+return                      RET(1!==1)  RET(2!==1*2)
+
+// factorial
+factorial_0:               *>0         *>0          *>2,0,1,2
```

Comment:
I'd prefer if the commentary to the right is behind `//` so this file is still syntacticly valid

Reply:
👍